web-wallet/src/lib/containers/MigrateContract/MigrateContract.svelte

```diff
@@ -0,0 +1,468 @@
+      icon={{ path: mdiWalletOutline }}
+      text={`CONNECT TO  ${selected === ERC_TOKEN ? "ETHEREUM" : "BSC"}`}
+      on:click={() => {
+        isWalletConnected = true;
+      }}
+    />
+  {:else if !isMigrationInitialized}
+    <Button
+      text="INITIALIZE MIGRATION"
+      on:click={() => {
+        isMigrationInitialized = true;
+      }}
+      disabled={!isAmountValid}
+    />
+  {/if}
+</article>
+
+<style lang="postcss">
+  .migrate {
+    border-radius: 1.25em;
+    background: var(--surface-color);
+    display: flex;
+    flex-direction: column;
+    gap: var(--default-gap);
+    padding: 1.25em;
+
+    &__header {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    &__header-icons {
+      display: flex;
+      align-items: center;
+      gap: 0.675em;
+    }
+
+    &__token,
+    &__amount {
+      display: flex;
+      flex-direction: column;
+      gap: var(--default-gap);
+      border-radius: 1.5em;
+      padding: 0.75em;
+      background-color: var(--background-color);
+    }
+
+    &__amount-token {
+      display: flex;
+      gap: var(--small-gap);
+      align-items: center;
+    }
+
+    &__progress-notice {
+      padding: 1em 1.375em;
+      border-radius: 1.5em;
+      border: 1px solid var(--primary-color);
+    }
+
+    &__token-header {
+      font-weight: 500;
+    }
+
+    &__token-address {
+      font-size: 1em;
+      font-weight: 500;
+      font-family: "Soehne Mono";
+      text-align: center;
+    }
+
+    &__token-balance {
+      display: flex;
+      justify-content: space-between;
+      font-size: 0.875em;
+
+      span {
+        font-weight: 500;
+        font-family: "Soehne Mono";
+      }
+    }
+
+    &__amount-header {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    &__amount-currency {
+      font-size: 0.875em;
+    }
+
+    &__amount-notice {
+      padding: 1em 1.375em;
+      border-radius: 0.675em;
+      border: 1px solid var(--error-color);
+      color: var(--error-color);
+    }
+
+    &__information-notice {
+      font-size: 0.875em;
+      line-height: 1.3125em;
+      padding: 1em 1.375em;
+      border-radius: 0.675em;
+      border: 1px solid var(--primary-color);
+      margin-top: 1.875em;
+    }
+
+    &__information-header {
+      display: flex;
+      justify-content: space-between;
+      padding-bottom: 1.25em;
+    }
+
+    &__information-time,
+    &__information-fee {
+      display: flex;
+      font-size: 0.875em;
+      align-items: center;
+      gap: var(--small-gap);
+
+      span {
+        display: flex;
+        align-items: center;
+      }
+    }
+
+    &__information-list {
+      padding-left: 1.375em;
+    }
+
+    &__information-approval {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: var(--default-gap);
+      padding: 2.25em 0;
+    }
+  }
+
+  :global(.dusk-textbox.migrate__input-field) {
+    background-color: var(--input--on-background-color);
```

Comment:
Why is this extra colour needed? Is this different from our other inputs?
If it is different I would make @laremas notice the discrepancy.

If the extra colour is needed for the time being anyway, we should give it a less confusing name. Something that better explains its purpose.

Reply:
The extra colour is needed as the design requires it.

Reply:
Aren't all our text inputs with a white background color in light mode and with a black one in dark mode already?